single_tech_samples/synapseanalytics/synapseserverless/README.md

```diff
@@ -0,0 +1,179 @@
+# Synapse Analytics Serverless Pools Best Practices using Delta Lake
+
+## Objectives
+
+This sample aims to demonstrate the use of Synapse Serverless Pools with Delta Lake files.
```

Comment:
I think the sample needs a bit more details to elaborate on the overall problem. The three objectives seems a bit "disjointed" with one talking about delta lake files, other one is automatic view creation and last Column level security -- but what is the overall problem that all these three things is trying to solve?